package/kedro_viz/server.py

```diff
@@ -474,6 +474,15 @@ def _get_dataset_data_params(namespace: str):
     return node_data
 
 
+def _get_parameter_values(node: Dict) -> Any:
+    """Get parameter values from a stored node."""
+    if node["obj"] is not None:
+        parameter_values = node["obj"].load()
+    else:  # pragma: no cover
```

Comment:
Adding `no cover` for now because it's very complicated to setup this test branch. I will check with @bru5 tomorrow to see if I can replicate his use case and maybe setup a better test.